doc/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
```

Comment:
Is it an architectural decision to have architectural decision records? 🤯

Reply:
This is automatically generated by [adr-tools](https://github.com/npryce/adr-tools) but yes!